clients.asciidoc

```diff
@@ -340,35 +340,64 @@ Geth and Parity can only operate fast synchronization when starting with an empt
+Trust Wallet:: A mobile Ethereum, Ethereum Classic wallet that supports ERC20, and ERC223 tokens. Trust Wallet is available for iOS and Android smartphones. Find it at https://trustwalletapp.com/
 
-=== Status
-Mobile browser in alpha. Decentralized all the way. Light client.
+=== Browser wallets
 
-=== Token
-Mobile browser in alpha. Pretty much the same as Status. A little more centralized to make operations easier for the user.
+A variety of wallets and DApp browsers are available as plugins or extensions of web browsers such as Chrome and Firefox: lightweight clients that run inside your browser.
 
-=== Trust DApp Browser
-Fully functioning Web3 browser that allows you to interact with decentralized applications (DApps) directly from the Trust Wallet. 
+Some of the more popular ones include MyEtherWallet/MyCrypto and MetaMask.
+
+==== MyEtherWallet (MEW)
+
+MyEtherWallet is a browser-based JavaScript lightweight client that offers:
+
+* A software wallet running in JavaScript
+* A bridge to popular hardware wallets such as the Trezor and Ledger
+* An RCP client that can connect to an Ethereum full client
+* A basic interface that can interact with smart contracts, given a contract's address and Application Binary Interface (ABI).
+
+MyEtherWallet is very useful for testing and as an interface to hardware wallets. It should not be used as a primary software wallet, as it is exposed to threats via the browser environment and is not a secure key storage system.
+
+You must be very careful when accessing MyEtherWallet and other browser-based JavaScript wallets, as they are frequent targets for phishing. Always use a bookmark and not a search engine or link to access the correct web URL. MyEtherWallet can be found at:
+
+https://MyEtherWallet.com
+
+
+==== MyCrypto
+
+Just prior to publication of the first edition of this book, the  MyEtherWallet project split into two competing implementations, guided by two independent development teams: a "fork" as it is called in open source development. The two projects are called MyEtherWallet (the original branding) and MyCrypto. At the time of the split, MyCrypto offered identical functionality as MyEtherWallet. It is likely that the two projects will diverge as the two development teams adopt different goals and priorities.
+
+As with MyEtherWallet, you must be very careful when accessing MyCrypto in your browser. Always use a bookmark, or type the URL very carefully (then bookmark it for future use).
+
+MyCrypto can be found at:
+
+https://MyCrypto.com
+
+==== MetaMask
+
+MetaMask was introduced in <<intro>>, and is a versatile browser-based wallet and basic contract explorer. It is available on Chrome, Firefox, Opera and Brave Browser. Find MetaMask at https://metamask.io
```

Comment:
For me, a remarkable aspect here is that **"Metamask"** is an **injector** of "a **web3 object** unlike the two cases previously exposed